app/views/user_sessions/_form.html.slim

```diff
@@ -0,0 +1,17 @@
+.login_page
```

Comment:
the files were created automatically like this from the tutorial I followed. 
I have changed to form_for(:session, url: user_sessions_path) in app/views/user_sessions/_form.html.slim from form_for(:session, url: login_page). Anywhere else changes are required for POST ?